client/src/Components/CommonComponent/Header/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// eslint-disable-next-line import/no-unresolved
+import './style.css';
+
+// eslint-disable-next-line react/prop-types
+export default function Header({ title, icon }) {
+  return (
+    <div className="header">
+      {icon}
+      <h2>{title}</h2>
+      <hr className="line" />
+    </div>
+  );
+}
```

Comment:
I expect the Eslint give you an error say `title is missing in props validation` and also for `icon is missing in props validation`
don't forget to add validation for it see this link https://reactjs.org/docs/typechecking-with-proptypes.html